Doc/library/statistics.rst

```diff
@@ -559,6 +570,93 @@ However, for reading convenience, most of the examples show sorted sequences.
 
    .. versionadded:: 3.8
 
+.. function:: covariance(x, y, /)
+
+   Calculates covariance of two variables *x* and *y*. Covariance is
+   a measure of the joint variability of two variables.
+
+   Raises :exc:`StatisticsError` if both variables have different number of data
```

Comment:
This can be reworded to be shorter and simpler, in keeping with the style of this document. For example:

"Both inputs must be of the same length (no less than two), otherwise StatisticsError is raised."

The same goes for the other two functions.